### PR TITLE
i3-wk-switch: init at 484f840

### PR DESCRIPTION
**Pull request by Synthetica9**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/window-managers/i3/wk-switch.nix [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub, python2Packages }:
+
+let
+  pkgName = "i3-wk-switch";
+in
+python2Packages.buildPythonApplication rec {
+  name = "${pkgName}-${version}";
```
Comment (grahamc) [inline pkgs/applications/window-managers/i3/wk-switch.nix]: Instead of the hash, please instead  use the date of the commit.

Comment (Synthetica9) [reply pkgs/applications/window-managers/i3/wk-switch.nix]: @grahamc Like this?

Comment (grahamc) [reply pkgs/applications/window-managers/i3/wk-switch.nix]: Perfect.

Comment (teto) [reply pkgs/applications/window-managers/i3/wk-switch.nix]: You can remove the name attribute, buildPythonApplication will concat pname and version attributes. As your set is recursive (rec) you can move pkgName into the set and rename it to `pname`

**Comment by Synthetica9** [comment]
@grahamc Bump

**Pull request merged**